frontend/src/components/file-preview-modal.tsx

```
@@ -0,0 +1,198 @@
+    };
+
+    const getFileTypeLabel = () => {
+        return fileType.toUpperCase();
+    };
+
+    const renderContent = () => {
+        switch (fileType) {
+            case 'html':
+                const sandboxPermissions = localEnableScripts
+                    ? "allow-same-origin allow-scripts"
+                    : "allow-same-origin";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)